src/components/CompanyForm.js

```diff
@@ -0,0 +1,42 @@
+//Dependencies
+import React, {PureComponent} from 'react'
+import {Redirect} from 'react-router-dom'
+import PropTypes from 'prop-types'
+
+class CompanyForm extends PureComponent {
+	state = {}
+
+	handleSubmit = (e) => {
+		e.preventDefault()
+		this.props.onSubmit(this.state)
+	}
+
+  handleChange = event => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value
+    });
+  };
+
+  render() {
+  		return (
+  			<form onSubmit={this.handleSubmit}>
```

Comment:
I know a way to make a form prettier - Material UI 😋 
so for styling this I would recommend to use it. (you can have a look for the login form for that for example)

Reply:
perfect that is what I meant by making it more stylish... Wanted first functionality... Thank you will do :smile: